### ISSUE TITLE
Could we have this repo synced with the copy in sage?

### ISSUE THREAD

**Issue by kiwifb** [miguelmarco/libhomfly#6]
Namely neither master or the v1.0 tarball have the fix for the `homfly.h` header. If one uses the version in this repo, the sage binding fail to build since it miss a definition for `Link`.

**Comment by miguelmarco** [comment]
Do you mean commit 02157faa9747bdccbc7dfcbf6e0f2f1b2c305f27 ?  It should be included in the tarball uploaded to the sage trac.

**Comment by kiwifb** [comment]
No. I am packaging sage for Gentoo and I tried to add libhomfly to the potential optional package for sage-on-gentoo and compilation failed. Because the version of `homfly.h` in this repo doesn't include the definition for the `Link` structure. The tarball I found on sage trac, which I understand is a different one from the one vanilla sage tries to download and which fails the checksum, has a different `homfly.h` file which includes

```
struct Link
{
    crossing *data;
    int num_crossings;
};
typedef struct Link Link;
```
